.github/workflows/deploy-to-production.yml

```
@@ -68,8 +78,8 @@
       - name: Download digests 📩
         uses: actions/download-artifact@v4
         with:
-          name: digests
-          path: /tmp/digests
+          name: docker-digest-${{ matrix.tag }}-*
```

actionlint

property "tag" is not defined in object type {}

(expression)